backend/prompt_studio/prompt_studio_core/views.py

```diff
@@ -393,43 +393,51 @@ def fetch_contents_ide(self, request: HttpRequest, pk: Any = None) -> Response:
         file_name: str = document.document_name
         view_type: str = serializer.validated_data.get("view_type")
 
+        # Extract filename without extension
         filename_without_extension = file_name.rsplit(".", 1)[0]
+
+        # Handle view_type logic, always converting to .txt for EXTRACT and SUMMARIZE
         if view_type == FileViewTypes.EXTRACT:
             file_name = (
                 f"{FileViewTypes.EXTRACT.lower()}/" f"{filename_without_extension}.txt"
             )
-        if view_type == FileViewTypes.SUMMARIZE:
+        elif view_type == FileViewTypes.SUMMARIZE:
             file_name = (
                 f"{FileViewTypes.SUMMARIZE.lower()}/"
                 f"{filename_without_extension}.txt"
             )
 
-        file_path = file_path = FileManagerHelper.handle_sub_directory_for_tenants(
+        file_path = FileManagerHelper.handle_sub_directory_for_tenants(
             UserSessionUtils.get_organization_id(request),
             is_create=True,
             user_id=custom_tool.created_by.user_id,
             tool_id=str(custom_tool.tool_id),
         )
-        file_system = LocalStorageFS(settings={"path": file_path})
+
+        # Ensure file path formatting
         if not file_path.endswith("/"):
             file_path += "/"
         file_path += file_name
-        # Temporary Hack for frictionless onboarding as the user id will be empty
```

Comment:
@jagadeeswaran-zipstack any reason why this comment was removed?

Reply:
This is from Remote storage came in when merging to main